ci/scripts/multi-arch-docker.sh

```diff
@@ -53,6 +55,25 @@ function pushDockerhub() {
   docker manifest push --insecure "$DOCKERTAG"
 }
 
+function isStableVersion() {
+    local version=$1
+    if [[ $version =~ ^v[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+        return 0  # Stable version
+    else
+        return 1  # Not a stable version
+    fi
+}
+
+function isLatestVersion() {
+    local version=$1
+    latest_version=$(git tag -l --sort=-v:refname | egrep "^v[0-9]+\.[0-9]+\.[0-9]+$" | head -n 1)
```

Comment:
Is is reliable to use `git tag` here? At least it seems we didn't pull tags here before the operation. 

Perhaps relying on curl github/dockerhub can be better.

Reply:
Added fetch tags before use. `curl` need `jq` to deal with the response. I prefer to use git here.